styles.css

```diff
@@ -446,6 +446,122 @@ textarea:focus-visible {
   margin-bottom: 200px;
 }
 
+.popup-card-close-btn-container {
+  display: flex;
+  justify-content: flex-end;
+}
+
+.popup-card-close-btn {
+  fill: #93a79d;
+  position: absolute;
+  top: 20px;
+  padding-right: 12px;
+}
+
+.popup-card-close-btn:hover {
+  transform: scale(1.4);
+  transition-duration: 0.4s;
+}
+
+.popup-card-close-btn:not(:hover) {
+  transition-duration: 0.4s;
+}
+
+.popup-card-container {
+  position: fixed;
+  background-color: #fcfcfc;
+  width: 100%;
+  height: 100%;
+  justify-content: center;
+  align-items: center;
+}
+
+.popup-card {
+  width: 90%;
+  height: 95%;
+  display: flex;
+  flex-direction: column;
+}
+
+.popup-card-text{
+  font: 600 25px/30px Inter, sans-serif;
+  color: var(--text-color-blue);
+  width: 90%;
+  height: 28%;
+
+}
+
+.popup-card-image-container {
+  height: 30%;
+}
+
+.popup-card-image{
+  width: 100%;
+  height: 100%;
+
+}
+
+.project-card-details {
+  height: 70%;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+}
+
+.popup-card-projects {
+  display: flex;
+  height: 15%;
+}
+
+.popup-card-projects li {
```

Comment:
And also compound styling tends to complicate someone's code, let give the li its own class and style the class

Reply:
> And also compound styling tends to complicate someone's code, let give the li its own class and style the class

yh i just updated this